core/src/dird/inc_conf.cc

```diff
@@ -860,7 +860,7 @@ static void StoreFname(LEX* lc,
                     lc->str);
           return;
         }
-        break;
```

Comment:
This should be no fallthrough in case if (strchr(lc->str, '\\')) { from line 846 ist false.